packages/google_maps_flutter/google_maps_flutter/example/ios/RunnerTests/PartiallyMockedMapView.h

```diff
@@ -0,0 +1,13 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import GoogleMaps;
+
+// Defines a map view used for testing key-value observing.
+@interface PartiallyMockedMapView : GMSMapView
+
+/// Counts the number of times that `frame` KVO has been added
```

Comment:
Same. You're also missing the period; comments should be properly punctuated, per the style guide.

Reply:
Also, this is a property, so should describe a noun rather than a verb; methods do things, properties are things. I.e., "The number of times that [...]".